x-pack/plugins/timelines/public/components/t_grid/toolbar/bulk_actions/alert_status_bulk_actions.tsx

```diff
@@ -121,7 +123,9 @@ export const AlertStatusBulkActionsComponent = React.memo<StatefulAlertStatusBul
           case 'in-progress':
             title = i18n.IN_PROGRESS_ALERT_FAILED_TOAST;
         }
-        addError(error.message, { title });
+        if (title) {
```

Comment:
same as above, I think you can remove this conditional and add 'acknowledged' to the switch statement